src/components/overview/Histogram.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+    BarChart,
+    Bar,
+    XAxis,
+    YAxis,
+    Tooltip,
+    Label,
+} from "recharts";
+
+class Histogram extends React.Component {
+  static propTypes = {
```

Comment:
this file has an indent size of 2 in some places and 4 in other places

Reply:
Fixed!